roles/validations/tasks/edpm/hugepages_and_reboot.yml

```diff
@@ -33,7 +33,7 @@
   cifmw.general.ci_script:
     output_dir: "{{ cifmw_validations_basedir }}/artifacts"
     script: >-
-      set -o pipefail && oc get -n {{ cifmw_validations_namespace }} osdpns "{{ deployed_nodeset_name.stdout }}" -o yaml | yq .status.configHash
```

Comment:
why do you remove the namespace?

Reply:
Ah, I can re-add this. I thought that was only in my DNM commit. Fixing it now.